elixir/apps/api/test/api/controllers/identity_controller_test.exs

```diff
@@ -88,7 +88,70 @@ defmodule API.IdentityControllerTest do
       assert json_response(conn, 401) == %{"error" => %{"reason" => "Unauthorized"}}
     end
 
-    test "returns a single resource", %{conn: conn, account: account, actor: actor} do
+    test "returns a single resource with populated email field", %{
```

Comment:
```suggestion
    test "returns a single identity with populated email field", %{
```

Reply:
All of the test names should be updated with a correct message now